src/Tools/DynamoFeatureFlags/App.config

```diff
@@ -1,9 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8" ?>
-<configuration>
-  <appSettings>
```

Comment:
this is going to cause us problems with point updates in our hosts - I think you'll need to keep these around.
@QilongTang @jasonstratton I am assuming we 2.19 at least will need to keep all files as is for .net48 builds?

Reply:
Just this one, ported the file to json